README.md

```diff
@@ -76,6 +65,26 @@ A less-recommended approach is to use the built-in [docker-ssh-env-config](https
 
 Another approach is to use the [environment agent hook](https://buildkite.com/docs/agent/hooks) to pull down the key into the container’s file system before the `git checkout` occurs. Note: the key will exist in Docker’s file system unless it is destroyed.
 
+## Advanced: docker-compose
+
+There is a wrapper script in the container that invokes a container for docker-compose. This is done to keep the image small, as docker-compose requires a python environment.
+
+Using it requires that you mount a volume into the container for `/buildkite`, and pass in the name of that volume as an env called ``.
```

Comment:
Oh this needs to be updated. I ended up going back to installing docker-compose directly. Adds about 90M to the image, but I think it's worth it.